src/plugins/data_source_management/public/components/data_source_selectable/data_source_selectable.scss

```diff
@@ -0,0 +1,12 @@
+.dataSourceSelectableOuiPanel {
+  width: 300px;
+
+  .dataSourceSelectableOuiFlexGroup {
+    .dataSourceSelectableOuiFlexItem {
+      text-overflow: ellipsis;
+      overflow: hidden;
+      white-space: nowrap;
+      display: inline-block;
+    }
+  }
+}
```

Comment:
Should we change the name to _data_source_selectable.cscc as a underscore _ shows this is a internal style file

Reply:
We should follow the dev guide `Do not use the underscore _ SASS file naming pattern when importing directly into a javascript file.` in https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#sass-files